source/opt/inst_bindless_check_pass.h

```diff
@@ -144,6 +145,11 @@ class InstBindlessCheckPass : public InstrumentPass {
   // Generate index of last byte referenced by buffer reference |ref|
   uint32_t GenLastByteIdx(ref_analysis* ref, InstructionBuilder* builder);
 
+  // Clone original original reference encapsulated by |ref| into |builder|.
+  // This may generate more than one instruction if neccessary.
```

Comment:
Clone original image computation starting at |image_id| into |builder|. Computation is part of larger descriptor-based reference |ref|.

Reply:
Actually ref is not needed here. I will remove the comment regarding ref.